backend/src/zimfarm_backend/db/models.py

```diff
@@ -226,6 +226,41 @@ class Schedule(Base):
         back_populates="schedule", cascade="all", init=False
     )
 
+    history_entries: Mapped[list["ScheduleHistory"]] = relationship(
+        back_populates="schedule",
+        cascade="all, delete",
+        passive_deletes=True,
+        init=False,
+        default_factory=list,
+        # return the history entries in descending order of created_at
+        order_by="ScheduleHistory.created_at.desc()",
+    )
+
+
+class ScheduleHistory(Base):
+    __tablename__ = "schedule_history"
+    id: Mapped[UUID] = mapped_column(
+        init=False, primary_key=True, server_default=text("uuid_generate_v4()")
+    )
+    schedule_id: Mapped[UUID] = mapped_column(
+        ForeignKey("schedule.id", ondelete="CASCADE"), init=False
+    )
+    author: Mapped[str]
+    created_at: Mapped[datetime] = mapped_column(index=True)
```

Comment:
Why all these indexed columns? I don't feel that we will heavily perform search filtering on these columns, besides `schedule_id`